src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -62,7 +62,7 @@ export default class DataTable extends Vue {
   treeSelection = Object.prototype.hasOwnProperty.call(this.$props.config, 'treeSelection')
     ? this.$props.config.treeSelection
     : defaultConfig.treeSelection;
-  printMode = this.$props.config.printMode || defaultConfig.printMode;
+  printMode = this.$props.config?.print?.mode || defaultConfig.print?.mode;
```

Comment:
```suggestion
  printMode = this.$props.config?.print?.mode || defaultConfig.print.mode;
```
We control the `defaulctConfig` so it must be set always 😉